Source/DOM classes/Unported or Partial DOM/SVGImageElement.m

```diff
@@ -105,7 +106,7 @@ - (CALayer *) newLayer
 	 */
 	AppleNativeImageRef image = [AppleNativeImage imageWithData:imageData];
 	
-    if( image == nil ) // NSData doesn't contain an imageformat Apple supports; might be an SVG instead
+    if( image == nil && imageURL ) // NSData doesn't contain an imageformat Apple supports; might be an SVG instead
```

Comment:
[SVGKSourceURL sourceFromURL:imageURL] will crash on iOS 8 if imageURL is nil, so I check imageURL at the line `if( image == nil )`.